dnn-tuning-nvidia-mxnet/cnn_text/graph.py

```diff
@@ -0,0 +1,74 @@
+import sys
+
+import mxnet as mx
```

Comment:
this import can take a long time... maybe we want a `print "Initializing..."` before all the imports, so users aren't frustrated by nothing happening